components/molecules/Searchbar/index.js

```diff
@@ -19,6 +19,10 @@ const SearchBar = ({ doFetchEvent }) => {
   const handleClick = () => {
     const sendCountry = location.country.toLowerCase().replace(' ', '-');
     const sendCity = location.city === 'Abuja Federal Capital Territory' ? 'abuja' : location.city.toLowerCase().replace(' ', '-');
+    localStorage.setItem('userSearchCountry', sendCountry);
+    localStorage.setItem('userSearchCity', sendCity);
+    localStorage.setItem('userEventType', interest);
+    console.log(sendCity, sendCountry, interest)
```

Comment:
console may not be needed anymore and can be safely removed??

Reply:
Yes. An apparent oversight. Thanks!